### PR TITLE
added starterkit

### PR DESCRIPTION
**Pull request by jairo-campos-JD**
Added starter kit

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+We're really glad you're reading this, because we need volunteer developers to help this project come to fruition. 👏
```
Comment (james-martin-jd) [inline CONTRIBUTING.md]: Replace with our CONTRIBUTING file

File: code-of-conduct.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
+# Contributor Covenant Code of Conduct
```
Comment (james-martin-jd) [inline code-of-conduct.md]: Does the license require us to keep this file? If not, let's remove it for now

File: tools/gh-pages-publish.ts [hunk 3]
```diff
@@ -0,0 +1,31 @@
+const { cd, exec, echo, touch } = require("shelljs")
```
Comment (james-martin-jd) [inline tools/gh-pages-publish.ts]: Are we planning to use gh-pages for anything?

File: tools/semantic-release-prepare.ts [hunk 4]
```diff
@@ -0,0 +1,54 @@
+const path = require("path")
```
Comment (james-martin-jd) [inline tools/semantic-release-prepare.ts]: Do we need this?

File: CONTRIBUTING.md (hunk 1 above)
Comment (jairo-campos-JD) [reply CONTRIBUTING.md]: updated

File: code-of-conduct.md (hunk 2 above)
Comment (jairo-campos-JD) [reply code-of-conduct.md]: removed it

File: tools/gh-pages-publish.ts (hunk 3 above)
Comment (jairo-campos-JD) [reply tools/gh-pages-publish.ts]: no, I deleted.

File: tools/semantic-release-prepare.ts (hunk 4 above)
Comment (jairo-campos-JD) [reply tools/semantic-release-prepare.ts]: no, I deleted it.

**Pull request merged**